plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/ElkUtil.java

```diff
@@ -895,8 +892,7 @@ public static void configureWithDefaultValues(final ElkPort port) {
      *            an edge of a graph
      */
     public static void configureWithDefaultValues(final ElkEdge edge) {
-        EdgeLabelPlacement elp = edge.getProperty(CoreOptions.EDGE_LABELS_PLACEMENT);
-        if (elp == EdgeLabelPlacement.UNDEFINED) {
+        if (!edge.hasProperty(CoreOptions.EDGE_LABELS_PLACEMENT)) {
             edge.setProperty(CoreOptions.EDGE_LABELS_PLACEMENT, EdgeLabelPlacement.CENTER);
```

Comment:
Is this necessary given that the default value is `CENTER`?

Reply:
As discussed, I didn't want to change the previous behavior: setting the value explicitly. We could change it in the future.